extensions/SiteMapIndex/Extension.pm

```diff
@@ -88,8 +88,6 @@ sub install_before_final_checks {
     print STDERR get_text('sitemap_requirelogin'), "\n";
     return;
   }
-
-  return if (Bugzilla->localconfig->urlbase ne 'https://bugzilla.mozilla.org/');
```

Comment:
This whole sub is a no-op and should be deleted.

Prior to [Bug 1386336](https://github.com/mozilla-bteam/bmo/pull/194/files#diff-8dbb7ef93c864e1fe9c0cca78918404459e5b85d93bf4dfb221dc345d6852f8c) there was a call to `$self->_fix_robots_txt();` at the end of the sub.